frontend/src/components/Home/NavBar.tsx

```diff
@@ -25,43 +25,43 @@ const NavBar= (): ReactElement => {
   const displayMobileMenu = (): ReactElement => {
     return (
       <Dropdown isOpen={toggleMenu} toggle={clickToggle}>
-      <DropdownToggle>
-        <span className="navbar-toggler-icon"></span>
-      </DropdownToggle>
-      <DropdownMenu right>
-        <DropdownItem><Link className="links" to="/faq">FAQ</Link></DropdownItem>
-      </DropdownMenu>
-    </Dropdown>
-  )
+        <DropdownToggle>
+          <span className="navbar-toggler-icon"></span>
+        </DropdownToggle>
+        <DropdownMenu right>
+          <DropdownItem><Link className="links" to="/faq">FAQ</Link></DropdownItem>
+        </DropdownMenu>
+      </Dropdown>
+    )
   }
 
   const displayNavBar = (): ReactElement => {
     return (
       <ul className='nav nav-button-margin' >
-              <li>
-                <button type="button" className="btn btn-lg btn-outline-dark" > 
-                <Link to='/faq' className="links">FAQ</Link>
-                </button>
-              </li>
+        <li>
+          <button type="button" className="btn btn-lg btn-outline-dark" >
+            <Link to='/faq' className="links">FAQ</Link>
+          </button>
+        </li>
       </ul>
-  )
+    )
   }
 
   return (
     <div>
-    <nav className="navbar navbar-expand-lg navbar-light">
-      <a className="navbar-brand" href="/">
-      <h1><img className="logo" src={icon} width="40" height="auto" alt="home icon"/> CU Housing</h1>
-      </a>
+      <nav className="navbar navbar-expand-lg navbar-light">
+        <a className="navbar-brand" href="/">
+          <h1><img className="logo" src={icon} width="40" height="auto" alt="home icon" /> CU Housing</h1>
+        </a>
 
-      {width > 600 ? displayNavBar() : displayMobileMenu()}
-      
-    </nav>
-    
-    <div className="homepage-description">
-    <h5>Search for off-campus housing, review apartments, and share feedback!</h5>
-  </div>
-  </div>
+        {width > 600 ? displayNavBar() : displayMobileMenu()}
```

Comment:
Not a problem with this PR, but we should eventually refactor this width state to be a hook we can use throughout the entire app (we also see it in `FAQNavBar`). Same with using axios everywhere - we should just have a general util file where we can grab this data from so we don't have to re-implement it every time.